app/assets/javascripts/utilities/buildArticleHTML.js

```diff
@@ -112,14 +112,30 @@ function buildArticleHTML(article, currentUserId = null) {
     var reactionsText = reactionsCount === 1 ? 'reaction' : 'reactions';
 
     if (article.class_name !== 'User' && reactionsCount > 0) {
-      reactionsDisplay =
-        '<a href="' +
-        article.path +
-        '"' +
-        commentsAriaLabelText +
-        'class="crayons-btn crayons-btn--s crayons-btn--ghost crayons-btn--icon-left"><svg class="crayons-icon" width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M18.884 12.595l.01.011L12 19.5l-6.894-6.894.01-.01A4.875 4.875 0 0112 5.73a4.875 4.875 0 016.884 6.865zM6.431 7.037a3.375 3.375 0 000 4.773L12 17.38l5.569-5.569a3.375 3.375 0 10-4.773-4.773L9.613 10.22l-1.06-1.062 2.371-2.372a3.375 3.375 0 00-4.492.25v.001z"/></svg>' +
-        reactionsCount +
-        `<span class="hidden s:inline">&nbsp;${reactionsText}</span></a>`;
+      var icons = [];
+      for (var category of article.public_reaction_categories) {
+        var icon = `<img src="/assets/${category.icon}.svg" width="18" height="18" />`;
+        icons = icons.concat(
+          `<span class="crayons_icon_container">${icon}</span>`,
+        );
+      }
+      icons.reverse();
+
+      reactionsDisplay = `<a href="${
+        article.path
+      }" class="crayons-btn crayons-btn--s crayons-btn--ghost crayons-btn--icon-left" data-reaction-count data-reactable-id="${
+        article.id
+      }">
+                          <div class="multiple_reactions_aggregate">
+                            <span dir="rtl">
```

Comment:
```suggestion
                            <span class="multiple_reactions_icons_container" dir="rtl">
```

This should make the stack UI correct and will show first reaction on top and last on bottom.

Reply:
Can you help me figure out what this class adds?

Without `multiple_reactions_icons_container`:

<img width="664" alt="Screen Shot 2023-02-24 at 11 54 10" src="https://user-images.githubusercontent.com/2077/221161657-3519b327-1791-449f-9ec1-61e35846c475.png">

With `multiple_reactions_icons_container`:

<img width="665" alt="Screen Shot 2023-02-24 at 11 54 33" src="https://user-images.githubusercontent.com/2077/221161714-36c87145-8868-4585-bd30-5b2637bad64e.png">

Reply:
I am not sure what the issue is but for me if I don't add this class in icon stack the right-most item is on top.

I am not blocking this PR on these changes but this will need to be discussed later.

Reply:
@rt4914 I figured it out! Browsers are (still) not the same — my screenshot above was in Firefox, but I had something very different in Chrome. With `multiple_reactions_icons_container`, both browsers now look the same as pictured.